test/spec/controllers/groupCtrlSpec.js

```diff
@@ -23,51 +23,207 @@ describe('Groups Controller', function() {
+         type: 'party',
+         members: ['leader-id'],
+         quest: {
+           progress: {},
+           active: true,
+           members: {}
+         }
+      });
+      pendingQuestParty= specHelper.newGroup({
+         _id: "unique-party-id",
+         type: 'party',
+         members: ['leader-id'],
+         quest: {
+           progress: {},
+           active: false,
+           members: {}
+         }
       });
+      runningQuestParty.quest.members[user._id] = true;
+      pendingQuestParty.quest.members[user._id] = true;
+    });
+ 
+    it('should return false if no quest in the group', function() {
+      runningQuestParty.quest = undefined;
+      expect(scope.isMemberOfPendingQuest(user._id, runningQuestParty)).to.eql(false);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(false);
+    });
 
-      expect(scope.isMemberOfGroup(user._id, guild)).to.be.ok;
-      expect(myGuilds).to.be.called;
+    it('should return false if no members in the group quest', function() {
+      runningQuestParty.quest.members = {};
+      expect(scope.isMemberOfPendingQuest(user._id, runningQuestParty)).to.eql(false);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(false);
     });
 
-    it('does not return true if guild is not included in myGuilds call', function(){
+    it('should return true/false depending on active flag and the method called', function() {
+      expect(scope.isMemberOfPendingQuest(user._id, runningQuestParty)).to.eql(false);
+      expect(scope.isMemberOfPendingQuest(user._id, pendingQuestParty)).to.eql(true);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(true);
+      expect(scope.isMemberOfRunningQuest(user._id, pendingQuestParty)).to.eql(false);
+    });
+
+    it('should return false if my user isn\'t part of the quest members', function() {
+      pendingQuestParty.quest.members = {};
+      runningQuestParty.quest.members = {};
+      expect(scope.isMemberOfPendingQuest(user._id, pendingQuestParty)).to.eql(false);
+      expect(scope.isMemberOfRunningQuest(user._id, runningQuestParty)).to.eql(false);
+    });
+  });
+   
+
+  describe("Group Edit Behavior", function() {
+
+    var guild, editGuild;
 
-      var guild = specHelper.newGroup({
+    beforeEach(function() {
+      guild = specHelper.newGroup({
         _id: "unique-guild-id",
         type: 'guild',
-        members: ['not-user-id']
+        members: ['not-user-id'],
+        $save: function() {}
       });
 
-      var myGuilds = sandbox.stub(groups,"myGuilds", function() {
-        return [];
-      });
+      // verify initial state, 
+      var editGuild = scope.groupCopy;
+      expect(editGuild).to.eql({});
+      expect(guild._editing).to.eql(undefined);
+    });
+
+
+    it('should allow for editing without changing group resource', function() {
+      // switch to edit mode
+      scope.editGroup(guild);
+      var editGuild = scope.groupCopy;
```

Comment:
Why is it important to assign this to a variable? Why can't you assert on `scope.groupCopy` directly?